meson.build

```diff
@@ -130,7 +130,7 @@ configure_file(
 	install_dir: datadir + '/dbus-1/services',
 )
 
-scdoc = dependency('scdoc', required: get_option('man-pages'), version: '>= 1.9.7')
+scdoc = dependency('scdoc', required: get_option('man-pages'), version: '>= 1.9.7', native: true)
 
 if scdoc.found()
 	sh = find_program('sh')
```

Comment:
We should also set `native: true` here. And in `protocol/meson.build` as well.

Reply:
Yeah, that's true. These places was `find_program` so I missed. Changed!